src/main/java/me/gnat008/perworldinventory/data/MySQL.java

```diff
@@ -0,0 +1,288 @@
+                    "`pid` INT NOT NULL," +
+                    "`items` BLOB NOT NULL," +
+                    "PRIMARY KEY (id)," +
+                    "FOREIGN KEY (pid) REFERENCES " + prefix + "players(pid));";
+            statement.executeUpdate(sql);
+
+            // Enderchest
+            sql = "CREATE TABLE IF NOT EXISTS `" + prefix + "enderchest` (" +
+                    "`id` INT NOT NULL AUTO_INCREMENT," +
+                    "`pid` INT NOT NULL," +
+                    "`items` BLOB NOT NULL," +
+                    "PRIMARY KEY (id)," +
+                    "FOREIGN KEY (pid) REFERENCES " + prefix + "players(pid));";
+            statement.executeUpdate(sql);
+
+            // Economy
+            sql = "CREATE TABLE IF NOT EXISTS `" + prefix + "economy` (" +
+                    "`id` INT NOT NULL AUTO_INCREMENT," +
+                    "`pid` INT NOT NULL," +
+                    "`bank_balance` DOUBLE," +
+                    "`balance` DOUBLE," +
+                    "PRIMARY KEY (id)," +
+                    "FOREIGN KEY (pid) REFERENCES " + prefix + "players(pid));";
+            statement.executeUpdate(sql);
+
+            // Stats
+            sql = "CREATE TABLE IF NOT EXISTS `" + prefix + "stats` (" +
+                    "`id` INT NOT NULL AUTO_INCREMENT," +
+                    "`pid` INT NOT NULL," +
+                    "`can_fly` BOOL NOT NULL," +
+                    "`display_name` VARCHAR(16) NOT NULL," +
+                    "`exhaustion` FLOAT NOT NULL," +
+                    "`exp` FLOAT NOT NULL," +
+                    "`flying` BOOL NOT NULL," +
+                    "`food` INT(20) NOT NULL," +
+                    "`health` DOUBLE NOT NULL," +
+                    "`level` INT(11) NOT NULL," +
+                    "`potion_effects` BLOB NOT NULL," +
+                    "`saturation` INT(20) NOT NULL," +
+                    "`fall_distance` FLOAT NOT NULL," +
+                    "`fire_ticks` INT NOT NULL," +
+                    "`max_air` INT NOT NULL," +
+                    "`remaining_air` INT NOT NULL," +
+                    "PRIMARY KEY (id)," +
+                    "FOREIGN KEY (pid) REFERENCES " + prefix + "players(pid));";
+            statement.executeUpdate(sql);
+
+            // Logout location
+            sql = "CREATE TABLE IF NOT EXISTS `" + prefix + "logout_location` (" +
+                    "`id` INT NOT NULL AUTO_INCREMENT," +
+                    "`pid` INT NOT NULL," +
+                    "`world` VARCHAR(255) NOT NULL," +
+                    "`x` DOUBLE NOT NULL," +
+                    "`y` DOUBLE NOT NULL," +
+                    "`z` DOUBLE NOT NULL," +
+                    "`pitch` FLOAT NOT NULL," +
+                    "`yaw` FLOAT NOT NULL," +
+                    "PRIMARY KEY (id)," +
+                    "FOREIGN KEY (pid) REFERENCES " + prefix + "players(pid));";
+            statement.executeUpdate(sql);
+
+            PwiLogger.info("MySQL tables created!");
```

Comment:
Nice code :heavy_check_mark: 

One thing to look into: are backticks supported by all DB types? I think this is a MySQL thing only.
`PwiLogger.info("MySQL tables created!");` will be logged each time the plugin starts :)